workflow/README.md

```diff
@@ -0,0 +1,20 @@
+## Workflow
+
+Attach a workflow, or series of commands, to a resource.
+
+```
+workflow(workflow_name : string, resource_name : string, work_cmds=[[]] : [[string]], clear_cmd=[]: [string])
```

Comment:
It might be helpful to include an example invocation here as guidance on the types, e.g. that `work_cmds` is an array of argv-style arrays

Reply:
Added. Thank you!